_tour/tuples.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: tour
+title: Tuples
+
+discourse: true
+
+partof: scala-tour
+
+num: 
+next-page: mixin-class-composition
+previous-page: traits
+topics: tuples
+
+redirect_from: "/tutorials/tour/tuples.html"
+---
+
+In Scala, a tuple is a class that can hold elements of different types.
```

Comment:
I'm never sure if you should say "is a class" or "is an object" in this case.

Reply:
It would be more correct to say "is a class whose instances hold elements". I think it's common enough to leave that implicit so as not to confuse people :-)